src/Styleguide/Components/AuctionResultItem.tsx

```diff
@@ -14,19 +14,14 @@ const ScaledArtworkImage = props => (
   </Flex>
 )
 
-export interface Dimensions {
-  in: string
-  cm: string
-}
-
 export interface AuctionResultItemProps {
   imageUrl: string
   title: string
   date: string
-  dimensions: Dimensions
+  dimensions: string
```

Comment:
Any reason the `Dimensions` interface was removed?

Reply:
This was actually after a discussion with @joeyAghion believe it or not!

Basically, in Gravity we have some code that takes our parsed version of auction lot data (in centimeters), and _also_ converts it into inches with fractional parts, (using `Math::Fractional` in Ruby), like `23 1/2`.

Since we're now going directly to Diffusion from Metaphysics, we still have access to the raw centimeter values, but we're losing this non-trivial fractional display/parsing.

However, since even our raw centimeter parsing can sometimes be inaccurate (and sometimes the inches conversion results in values like `23 1/10`), we decided in our new version maybe we should _only_ show the dimensions (as best we could identify), _exactly as they appear in the source document_.

Diffusion offers that, as `dimension_text`, as well as the centimeter values. So this PR/Relay query winds up just querying and rendering that field, vs any converted values.

If we do want to reinstate some sort of friendly fractional display, we can investigate possible JS solutions in Metaphysics (since we'll have raw centimeters), or we can port that from Gravity to Diffusion.

Reply:
Makes sense 👍